src/providers/elevenlabs_agent.py

```
@@ -364,44 +368,67 @@ async def send_interrupt(self) -> None:
             try:
-                await asyncio.wait_for(self._keepalive_task, timeout=1.0)
-            except (asyncio.CancelledError, asyncio.TimeoutError):
+                await self.on_event({
+                    "type": "session_ended",
+                    "call_id": self._call_id,
+                    "provider": "elevenlabs_agent",
+                    "audio_sent_bytes": self._session_state.total_audio_sent,
+                    "audio_received_bytes": self._session_state.total_audio_received,
+                })
+            except Exception:
                 pass
```

Ruff (0.15.0)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)